packages/compass-indexes/src/modules/search-indexes.ts

```diff
@@ -9,10 +9,8 @@ import type { SearchIndex } from 'mongodb-data-service';
 
 import { isAction } from '../utils/is-action';
 import { FetchStatuses, NOT_FETCHABLE_STATUSES } from '../utils/fetch-status';
-import type { FetchStatus } from '../utils/fetch-status';
 
 import { FetchReasons } from '../utils/fetch-reason';
-import type { FetchReason } from '../utils/fetch-reason';
```

Comment:
Would be nice if we could go the other route and remove the enums, is that an option here?